### ISSUE TITLE
Document which combinations of sectors can be modelled.

### ISSUE THREAD

**Issue by fneum** [PyPSA/pypsa-eur-sec#176]
We currently have sectors electricity, heat (H), industry (I), biomass (B), agriculture (A), transport (T).

Not all subsets are possible. For instance, I requires B (because some demands in paper and pulp industry can only be met by biomass).

The long-term vision would be to disentangle these dependencies completely where applicable.

**Comment by martavp** [comment]
Yes, this is needed. Another example is that currently, adding "T" without "I" produces an error if some of the vehicles remain ICE. I think the lines within the "if" statement are responsible:

https://github.com/PyPSA/pypsa-eur-sec/blob/23a167c308f438d10e4d5608f6457e3a1d46289a/scripts/prepare_sector_network.py#L1308
